### ISSUE TITLE
How many letters should there be in the alphabet?

### ISSUE THREAD

**Issue by rottenoats** [TerraVera/TerraVeraLang#2]
As mentioned in #3 , the number of letters should follow a "regularity" and I find that the powers of 2 something important to use.

As an example, there could be 32 or 64 letters in the alphabet.

**Comment by Zyfast** [comment]
Yeah, definitely, everything is based on 2. Let it be 32 though, not to make it too complicated. It's like Russian alphabet -1 letter hehe

**Comment by rottenoats** [comment]
Some primarly letters could be:

a, b, d, e, f, g, h, i, k ,l, m, n, o p, q, r, s, t, u, v, w, z

This is taking into account certain letters in the english alphabet are created from existing letters, which means they should be removed:

as "c" is simply "se" together and "x" is simply "ks"

This still leaves, 10 letters to determine as it should be either 16 or 32. What do you think?

**Comment by Zyfast** [comment]
Good idea about simplifying the Latin alphabet! It's very logical, I like it. I'd just eliminate q since you can write it like "k'u" (using the phonetical writing, you know). Otherwise, what if we use some cyrillic symbols? But not the ы, because it is difficult to pronounce. Um, for example, Э is a good sound because you can just add the "I" and it'll sound like Russian E. You see what I mean? But let me think it over
